test/runtests.jl

```diff
@@ -180,6 +180,25 @@ end
     @test GLM.dispersion(lm_model) ≈ GLM.dispersion(lm_qr_model)
 end
 
+@testset "Linear model with Cholesky dropcollinearity" begin
```

Comment:
Since this testset is identical to the one below for QR, why not use a single block, with a `for method in (:qr, :cholesky)`? Same for other sets.

Reply:
Updated most of the test cases.